### ISSUE TITLE
Multiple files of the same name

### ISSUE THREAD

**Issue by thehilll** [akrabat/rodeo#45]
I haven't actually run into this issue, but looking at `rodeo-uploaded-files.json` it seems to have only the file name, not the full path:

```
{
  "DSCF1493.jpg": "123..."
}
```

Looking at my photo library...I have multiple images that will have the same file name.  These are from different cameras years apart that use the same naming sequence.

Will there be name collisions where rodeo will think a file has been uploaded, but really it was a different file of the same name?

Thank you.

**Comment by akrabat** [comment]
Yes, there will be as it's not a clever system. You can choose to have it store the list of uploaded files in the directory where there the file is, so you could just delete the file so that rodeo forgets. 

It would make sense to have a setting to disable this feature though.

**Comment by thehilll** [comment]
Oh, but `store_uploaded_list_in_image_dir: true` would then avoid this problem.  That's great.  Thanks.
